### ISSUE TITLE
Crash occuring on Xcode 12 when attempting to do enum reflection on child action.

### ISSUE THREAD

**Issue by vibrazy** [pointfreeco/swift-composable-architecture#200]
**Describe the bug**
Attempting to send an `action` view the store causes `EXC_BAD_INSTRUCTION (code=EXC_I386_INVOP, subcode=0x0)` when combining reducers.

**Given** a child action without any cases
```enum ChildAction: Equatable {}```

**And a parent action**
```swift
enum ParentAction: Equatable {
	case childAction(ChildAction)
	case parentAction
}
```
**And a main reducer**
```swift
let parentReducer = Reducer<ParentState, ParentAction, ParentEnvironment>.combine(
	childReducer.pullback(
	 state: \.childState,
		action: /ParentAction.childAction,
		environment: { _ in ChildEnvironment() }
	),
	Reducer<ParentState, ParentAction, ParentEnvironment> { state, action, _ in
		switch action {
		case .parentAction:
			state.childState += 1
			return .none
		case .childAction:
			return .none
		}
	}
)
```
As soon as you send `viewStore.send(.parentAction)` a crash occurs.
The crash happens on `EnumReflection` line 75 from the CasePaths package

This crash does not happen on Xcode 11.4 with the same setup.

A work around is to add a case to the child enum.

```swift
enum ChildAction {
	case banana
}
```

**To Reproduce**
Send any action `viewStore.send(.parentAction)` with the above setup is sufficient to cause a crash.

**Expected behavior**
Give a clear and concise description of what you expected to happen.

**Screenshots**
<img width="1085" alt="Screen Shot 2020-06-26 at 15 56 53" src="https://user-images.githubusercontent.com/16307715/85871022-c16fee80-b7c5-11ea-81da-d0d0ece6f1dd.png">

**Environment**
 - Xcode 12 Beta 1
 - Swift 5.3
 - OS (if applicable): iOS 14

**Additional context**
[ComposableCasePathCrash.zip](https://github.com/pointfreeco/swift-composable-architecture/files/4837700/ComposableCasePathCrash.zip)

**Comment by stephencelis** [comment]
Hey @vibrazy, thanks for the detailed report. We've actually logged this issue on swift-case-paths here: https://github.com/pointfreeco/swift-case-paths/issues/11

We probably won't get around to fixing it for a bit (if you wanna take a pass at it, please do!), but in the meantime, another workaround is to use the `.never` case path:

```swift
 childReducer.pullback(
   state: \.childState,
-  action: /ParentAction.childAction,
+  action: .never,
   environment: { _ in ChildEnvironment() }
 ),
```

**Comment by stephencelis** [comment]
This has been fixed upstream in [Case Paths 0.1.2](https://github.com/pointfreeco/swift-case-paths/releases/tag/0.1.2). Be sure to update your package dependencies!
